Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs

```diff
@@ -41,7 +38,7 @@ public QuestionRepository(TrappistDbContext dbContext, UserManager<ApplicationUs
         {
             var question = Mapper.Map<QuestionDetailAC, Question>(questionAC.Question);
             var singleMultipleAnswerQuestion = Mapper.Map<SingleMultipleAnswerQuestionAC, SingleMultipleAnswerQuestion>(questionAC.SingleMultipleAnswerQuestion);
-            question.ApplicationUser = await _userManager.FindByEmailAsync(userEmail);
+            //To-Do Add created by user Id question.CreatedByUser = await _userManager.FindByEmailAsync(userEmail);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS125)